### PR TITLE
Adding some content to the README file

### PR DESCRIPTION
**Pull request by PeterBell**
Making it easier for people to understand what this project is about.

### CODE REVIEW

**Comment by zangchung** [comment]
@PeterBell This book is very useful

**Comment by Jiculi** [comment]
hola

**Comment by gleibson** [comment]
test

File: README.md [hunk 1]
```diff
@@ -1,2 +1,8 @@
 single-repo-example
 ===================
+
+Here is some new content for the README.md file
+
```
Comment (cafe3165) [inline README.md]: It 's a test from China

**Comment by Juliefornia** [comment]
Doing github tutorial. 'Edit readme.md without fork and pull request'
Couldn't edit readme.md tho, but pull request anyway.

**Comment by MilkXu** [comment]
good

**Comment by cafe3165** [comment]
你是😂


------------------ 原始邮件 ------------------
发件人: "MilkXu";<notifications@github.com>;
发送时间: 2016年10月26日(星期三) 下午2:55
收件人: "pragmaticlearning/single-repo-example"<single-repo-example@noreply.github.com>; 
抄送: "白月石"<78599297@qq.com>; "Comment"<comment@noreply.github.com>; 
主题: Re: [pragmaticlearning/single-repo-example] Adding some content tothe README file (#1)




good

**Comment by tkadjm** [comment]
Testing this.
:bowtie:

**Comment by juceyjing** [comment]
hahha

**Comment by brntbeer** [comment]
getting a new version of the book out with @PeterBell (I know, it's about time). Closing and deleting this branch! ❤️

**Comment by Jaremekgw** [comment]
If it's closed it's no possibility to put comments?

**Pull request closed without merging**